plugin/pom.xml

```diff
@@ -182,6 +182,7 @@
                 <artifactId>maven-plugin-plugin</artifactId>
                 <configuration>
                     <skipErrorNoDescriptorsFound>true</skipErrorNoDescriptorsFound>
+                    <goalPrefix>oc</goalPrefix>
```

Comment:
I suppose that in the future this should be taken into consideration in the documentation we need to do right?

Reply:
Yes, I would start documentation work after I'm done with migrating all plugins. I think maybe I would change artifactId to `oc-maven-plugin` instead of `openshift-maven-plugin`